tmt/steps/prepare/ansible.py

```diff
@@ -70,8 +70,11 @@ class PrepareAnsible(tmt.steps.prepare.PreparePlugin[PrepareAnsibleData]):
 
     .. warning::
 
-        When specifying playbooks with paths, all paths must be
-        relative to the metadata tree root.
+        When specifying playbooks with paths:
+        if metadata tree root exist:
+            all paths must be relative to the metadata tree root.
+        if metadata tree root NOT exist:
+            all paths must be relative to the current working directory.
```

Comment:
Is it readable and well structered enough?

Reply:
Hm, and for some reason, after I commited the pre-commit didnt start the tmt lint check
`tmt lint.............................................(no files to check)Skipped`

Reply:
Not bad at all, although there are small changes I'd make: some a bit of rewording, and we can use ReST formatting for better rendering (try it out with `tmt run prepare -h ansible --help` and/or `make docs`, `docs/_build/plugins/prepare.html#ansible`)

```rest
        When specifying playbooks with paths:

        * If a metadata tree root exists, all paths must be relative to the metadata tree root.
        * If the metadata tree root does not exist, all paths must be relative to the current working directory.
```